apps/meteor/app/livechat/server/api/v1/contact.ts

```diff
@@ -136,3 +141,23 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'omnichannel/contacts.search',
+	{ authRequired: true, permissionsRequired: ['view-livechat-contact'], validateParams: isGETOmnichannelContactsSearchProps },
+	{
+		async get() {
+			if (process.env.TEST_MODE?.toUpperCase() !== 'TRUE') {
+				throw new Meteor.Error('error-not-allowed', 'This endpoint is only allowed in test mode');
+			}
```

Comment:
What's the point of the endpoint if it can't be used?

Reply:
This will be removed once the feature is fully done, we've decided to put it under a "feature" flag in order to merging it directly into develop.